sdk/formrecognizer/Azure.AI.FormRecognizer/src/TrainingFileFilter.cs

```diff
@@ -13,5 +13,12 @@ public partial class TrainingFileFilter
         internal TrainingFileFilter()
         {
         }
+
+        /// <inheritdoc />
+        public string Prefix { get; set; } = string.Empty;
+
+        /// <inheritdoc />
+        public bool? IncludeSubFolders { get; set; } = false;
```

Comment:
Would it be possible to make this non-nullable while you're in there?
https://github.com/Azure/azure-sdk-for-net/issues/10990

Reply:
Ohh I missed that. 
Yes. On it

Reply:
Updated